### PR TITLE
Spelling and grammar fixes suggested by PyCharm code inspection

### PR DESCRIPTION
**Pull request by MyreMylar**
PyCharm has a spelling and grammar code inspection tool that you can run. 

I pointed it at `/src_c/`, `/src_py/` and `/test/` to start with and accepted the suggestions that seemed reasonable. This PR is the result.

### CODE REVIEW

File: test/draw_test.py [hunk 1]
```diff
@@ -2366,7 +2366,7 @@ def test_lines__surface_clip(self):
 
         clip_rect = pygame.Rect((0, 0), (11, 11))
         clip_rect.center = surface.get_rect().center
-        pos_rect = clip_rect.copy()  # Manages the lines's pos.
+        pos_rect = clip_rect.copy()  # Manages the lines' pos.
```
[the remark this answers is not in the document]
Comment (MyreMylar) [reply test/draw_test.py:2369]: It is multiple lines in this case as it is the `draw.lines()` function being tested.

File: test/test_utils/png.py [hunk 2]
```diff
@@ -2573,7 +2573,7 @@ def testLA4(self):
         self.assertEqual(sbit, strtobytes("\x04\x04"))
 
     def testPNMsbit(self):
-        """Test that PNM files can generates sBIT chunk."""
+        """Test that PNM files can generate sBIT chunk."""
```
[the remark this answers is not in the document]
Comment (MyreMylar) [reply test/test_utils/png.py]: thanks! fixed.

File: src_py/surfarray.py [hunk 3]
```diff
@@ -96,7 +96,7 @@
 def blit_array(surface, array):
     """pygame.surfarray.blit_array(Surface, array): return None
 
-    Blit directly from a array values.
+    Blit directly from an array values.
```
[the remark this answers is not in the document]
Comment (MyreMylar) [reply src_py/surfarray.py]: Thanks! fixed.

**Comment by MyreMylar** [comment]
> Also what about docs? Those also have a bunch of typos and whatnot.

I think I'll leave that to another PR on another day as this one is already fairly chunky with the code file fixes. 

There is also `/examples/`.

**Review by Matiiss — APPROVED** [review]
LGTM

**Review by Starbuck5 — APPROVED** [review]

**Pull request merged**